src/components/content/common/ocl/DisplayOclData.tsx

```diff
@@ -108,6 +109,11 @@ function DisplayOclData({ ocl }: { ocl: Ocl }): React.JSX.Element | string {
                                 />
                             </Descriptions.Item>
                             <Descriptions.Item label='Description'>{ocl.description}</Descriptions.Item>
+                            {ocl.eula ? (
```

Comment:
If EULA is not present -> we can display directly text as `Not Provided` instead of completely avoiding it.